packages/botonic-plugin-contentful/src/tools/validate-all-contents.ts

```diff
@@ -10,11 +11,13 @@ export class ContentsValidator {
    * - There are infinite loops
    * - There's a bug on the CMS plugin
    */
-  async validateAllTopContents(context: ContextWithLocale) {
-    await Promise.all(
-      TOP_CONTENT_TYPES.map(ct => {
-        return this.cms.topContents(ct, context)
-      })
+  async validateAllTopContents(context: ContextWithLocale): Promise<void> {
+    // topContents will internally limit concurrency
+    const concurrency = context.concurrency && 1
```

Comment:
Why this `&& 1`?

Reply:
we want to limit to 1 when not specified because the call to topContents will already execute in parallel